### ISSUE TITLE
Amid Evil dx12 issue solved, only issue is the log file 

### ISSUE THREAD

**Issue by washingleavesandthensome23-434** [doitsujin/dxvk#2506]
I somehow missed the whole Windows 7 thing.

Can we ***PLEASE*** stop with this kind of bullshit? D3D12 is not supported on Windows 7 unless the game explicitly ships the [12-on-7](https://devblogs.microsoft.com/directx/porting-directx-12-games-to-windows-7/) runtime, and vkd3d-proton will not change that since games tend to actually check the Windows version. Yes, exceptions where it *does* work may exist, but it's not the norm, nor expected, and by opening bug reports when running an *explicitly unsupported configuration* here just wastes everybody's time.

TL;DR if you want to run the game in D3D12 mode, install Windows 10.

_Originally posted by @doitsujin in https://github.com/doitsujin/dxvk/issues/2503#issuecomment-1046128864_

**Comment by washingleavesandthensome23-434** [comment]
so ive solved the issue ...by removing the dx12 support check flag , runs in dx12 now with said dxvk/d3dvk setup , BUT my log is abit ...big , if we could get this resolved , id be a happy camper

**Comment by washingleavesandthensome23-434** [comment]
https://www.mediafire.com/file/21z2w22iop89utp/AmidEvil-Win64-Shipping_patched_dxgi.log/file

https://www.mediafire.com/file/5fn5joyozq8cyvp/AmidEvil-Win64-Shipping_patched_d3d11.log/file

**Comment by washingleavesandthensome23-434** [comment]
also i ask here , because you guys are the pros when it comes to dxvk/d3dvk...

**Comment by doitsujin** [comment]
Which part of *please stop wasting our time* do you not understand?

**Comment by washingleavesandthensome23-434** [comment]
ok man , thought you could help a bro out , unsupported or not
